src/less/variables.less

```diff
@@ -80,6 +80,7 @@
 //
 
 
+@global-screen-xxsmall-min: 360px;
```

Comment:
@Sonam-R I added this comment in the rhinofront PR but adding here as well just in case. Let's name this breakpoint `nano` instead of `xxsmall` so that we avoid any confusion in the future with the existing `xsmall` breakpoint

Reply:
@juanfabrega Done..